src/rpc/rawtransaction.cpp

```diff
@@ -1259,6 +1259,415 @@ static UniValue testmempoolaccept(const JSONRPCRequest& request)
+            "      \"pubkey\" : \"pubkey\",                     (string) The public key this path corresponds to\n"
+            "      \"master_fingerprint\" : \"fingerprint\"     (string) The fingerprint of the master key\n"
+            "      \"path\" : \"path\",                         (string) The path\n"
+            "      }\n"
+            "    }\n"
+            "    ,...\n"
+            "  ],\n"
+            "  \"inputs_provided\" : n,       (numeric, optional) The number of inputs provided in the PSBT\n"
+            "  \"inputs\" : [                 (array of json objects)\n"
+            "    {\n"
+            "      \"non_witness_utxo\" : {   (json object, optional) Decoded network transaction for non-witness UTXOs\n"
+            "        ...\n"
+            "      },\n"
+            "      \"witness_utxo\" : {            (json object, optional) Transaction output for witness UTXOs\n"
+            "        \"value\" : x.xxx,            (numeric) The value in " + CURRENCY_UNIT + "\n"
+            "        \"n\" : n,                    (numeric) index\n"
+            "        \"scriptPubKey\" : {          (json object)\n"
+            "          \"asm\" : \"asm\",            (string) The asm\n"
+            "          \"hex\" : \"hex\",            (string) the hex\n"
+            "          \"reqSigs\" : n,            (numeric) The required sigs\n"
+            "          \"type\" : \"pubkeyhash\",    (string) The type, eg 'pubkeyhash'\n"
+            "          \"addresses\" : [           (json array of string)\n"
+            "            \"address\"               (string) Bitcoin address\n"
+            "            ,...\n"
+            "          ]\n"
+            "        }\n"
+            "      },\n"
+            "      \"partial_signatures\" : [             (array of json objects, optional)\n"
+            "        {\n"
+            "          \"pubkey\" : \"pubkey\",             (string) The public key this signature corresponds to\n"
+            "          \"signature\" : \"signature\",       (string) The signature\n"
+            "        }\n"
+            "        ,...\n"
+            "      ]\n"
+            "      \"sighash\" : \"type\",                  (string, optional) The sighash type recommended to be used\n"
+            "      \"index\" : n                          (numeric) The input index of this input\n"
+            "    }\n"
+            "    ,...\n"
+            "  ]\n"
+            "}\n"
+
+            "\nExamples:\n"
+            + HelpExampleCli("decodepsbt", "\"hexstring\"")
+            + HelpExampleRpc("decodepsbt", "\"hexstring\"")
+    );
+
+    RPCTypeCheck(request.params, {UniValue::VSTR});
+
+    // Unserialize the transactions
+    PartiallySignedTransaction psbtx;
+    if (!IsHex(request.params[0].get_str())) {
+        throw JSONRPCError(RPC_DESERIALIZATION_ERROR, "TX decode failed, not hex");
+    }
+    std::vector<unsigned char> txData(ParseHex(request.params[0].get_str()));
```

Comment:
Use `ParseHexV` instead of `IsHex` + `ParseHex`?

Reply:
Done